src/middle/Location.ml

```diff
@@ -58,18 +37,20 @@ Format the location for error messaging.
 If printed_filename is passed, it replaces the highest-level name and
 leaves the filenames of included files intact.
 *)
-let rec to_string ?printed_filename ?(print_file = true) ?(print_line = true)
+let rec pp ?printed_filename ?(print_file = true) ?(print_line = true) () ppf
```

Comment:
I find that extra `()` rather ugly but I guess it can't be helped. (Weird that a "functional programming language" would object to "partial function applications"...)

Reply:
Let me know if you like 008e78e more or less than the `()`